### ISSUE TITLE
Random Ctd

### ISSUE THREAD

**Issue by Supercrash9** [zhongjiedong/Tsukiro#82]
Hi, everyone,

first of all i love this modlist it feels like a new game not like a 10 year old one but it is sometimes almost unplayable for me the first 60 minutes after a new start are actually always good then the ctd start completely randomly no matter if inside or outdoors, whether in rifts, in solitude, white run or near falcon ring. for me they are not reproducible. After restarting the game or the computer, the same thing does not appear again in the same place. sometimes they are every 10 minutes sometimes can play through 30 minutes. if they were every 2-3 hours then it would be less of a problem but every 10 minutes is tiring. I followed the guide 1 to 1 during the installation and didn't add any mods or turn them off. except as described in point 4 the mods in the game. I hang the last 5 net. crash logs here. I hope someone can help me and it is something very simple that I have overlooked I would really like to play through the mod list once.
ps. reinstallation of the entire brought no success.
sorry for the bad english google translator i'm from germany

cya

[Crash_2022_2_1_22-38-7.txt](https://github.com/zhongjiedong/Tsukiro/files/7995966/Crash_2022_2_1_22-38-7.txt)
[Crash_2022_2_1_22-52-45.txt](https://github.com/zhongjiedong/Tsukiro/files/7995967/Crash_2022_2_1_22-52-45.txt)
[Crash_2022_2_1_23-2-26.txt](https://github.com/zhongjiedong/Tsukiro/files/7995968/Crash_2022_2_1_23-2-26.txt)
[Crash_2022_2_1_23-16-1.txt](https://github.com/zhongjiedong/Tsukiro/files/7995969/Crash_2022_2_1_23-16-1.txt)
[Crash_2022_2_2_19-1-51.txt](https://github.com/zhongjiedong/Tsukiro/files/7995970/Crash_2022_2_2_19-1-51.txt)

**Comment by YoYoK3nny** [comment]
In addition to the steps in the guide, did you also do the steps in the [readme](https://github.com/zhongjiedong/Tsukiro#readme)? Specifically the [pagefile](https://github.com/zhongjiedong/Tsukiro#pagefile-configuration) config?

While it won't apply to everyone, I've found that the main pagefile needs to also be at least a minimum of 20480MB (in my case the pagefile on the drive Skyrim is installed to never shows as allocated even after restarts, possibly due to being on a USB external HDD. As a fallback, I've set my main pagefile to the minimum plus 20000MB)

**Comment by Supercrash9** [comment]
Hi there,
Yes, I have carried out all the steps, including those from the readme, and I have now solved the problem. I tried it out for a few more days and ran something like memtest and various stability benchmarks over it. Other mod lists tried it out. memtest benchmarks didn't show any errors or issues either. Then I talked to a friend about it and he had a similar problem with a modded minecraft, yes I know it's not the same. but he said it helped to deactivate his xmp profile for the ram in the bios. and lo and behold it solved it. I don't know why is 3200mhz too fast for a 10 year old game or does the profile run stable enough for vanilla skyrim and other vanilla games like cyberpunk but slightly unstable for a heavily modded skyrim that uses 20 to 25 gigabytes of my 32 I know it doesn't..... conclusion now it's running 4 hours of play 6 hours of play 12 hours overnight my char stands around before white run not a single ctd at 40 to 55 fps on average without adjustments. but thanks for the answer. I'm looking forward to enjoying Skyrim with the mod list in peace. Greetings from Germany
